website/docs/docs/partnerships.mdx

```diff
@@ -9,38 +9,30 @@ description: >-
 
 The Terraform Integration Program facilitates prospect partners in creating and publishing Terraform integrations that have been verified by HashiCorp.
```

Comment:
```suggestion
The Terraform Integration Program facilitates prospect partners in creating and publishing Terraform integrations verified by HashiCorp.
```

Suggest changing "that have been verified" to "verified' to avoid a passive voice style issue.

Reply:
I'll change the verbiage to validated actually